plenum/server/consensus/consensus_shared_data.py

```diff
@@ -28,6 +28,8 @@ def __init__(self, name: str, validators: List[str], inst_id: int, is_master: bo
         self.inst_id = inst_id
         self.view_no = 0
         self.waiting_for_new_view = False
+        # TODO: Do we need primaries for all instances here?
```

Comment:
I believe we need all of them because all Primaries are stored in the Audit Ledger.